src/m365/spo/commands/group/group-user-add.ts

```diff
@@ -0,0 +1,111 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandError,
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  groupId: number;
+  userName: string;
+}
+
+class SpoGroupUserAddCommand extends SpoCommand {
+
+  public get name(): string {
+    return commands.GROUP_USER_ADD;
+  }
+
+  public get description(): string {
+    return 'Adds user to a SharePoint Group';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
+
+    if (this.verbose) {
+      logger.logToStderr(`Start adding user/s to SharePoint Group ${args.options.groupId}...`);
+    }
+
+    const data: any = {
+      url: `${args.options.webUrl}`,
+      peoplePickerInput: this.getFormatteduserList(args.options.userName),
+      roleValue: `group:${args.options.groupId}`
+    };
+
+    const requestOptions: any = {
+      url: `${args.options.webUrl}/_api/SP.Web.ShareObject`,
+      headers: {
+        'Accept': 'application/json;odata=nometadata',
+        'Content-type': 'application/json;odata=verbose'
+      },
+      data: data,
+      responseType: 'json'
+    };
+
+    request.post<string>(requestOptions)
+      .then((rawRes: any): void => {
```

Comment:
As we expect the returned object to have properties, let's define them on the returned type rather than using `any`, so that we can avoid errors in refactoring in the future.